index.html

```diff
@@ -0,0 +1,314 @@
+          <div class="slide-number">1 / 3</div>
+          <img
+            src="https://picsum.photos/id/1015/500/200"
+            style="width: 100%; object-fit: contain"
+          />
+          <div class="slide-text">Caption One</div>
+        </div>
+
+        <div class="slides fade">
+          <div class="slide-number">2 / 3</div>
+          <img
+            src="https://picsum.photos/id/1016/500/200"
+            style="width: 100%; object-fit: contain"
+          />
+          <div class="slide-text">Caption Two</div>
+        </div>
+
+        <div class="slides fade">
+          <div class="slide-number">3 / 3</div>
+          <img
+            src="https://picsum.photos/id/1039/500/200"
+            style="width: 100%; object-fit: contain"
+          />
+          <div class="slide-text">Caption Three</div>
+        </div>
+
+        <a class="prev" onclick="plusSlides(-1)">❮</a>
+        <a class="next" onclick="plusSlides(1)">❯</a>
+      </div>
+
+      <div class="mt-5" style="text-align: center">
+        <span
+          class="dot"
+          style="width: 10px; height: 10px"
+          onclick="currentSlide(1)"
+        ></span>
+        <span
+          style="width: 10px; height: 10px"
+          class="dot"
+          onclick="currentSlide(2)"
+        ></span>
+        <span
+          style="width: 10px; height: 10px"
+          class="dot"
+          onclick="currentSlide(3)"
+        ></span>
+      </div>
+    </section>
+    <!-- image slider -->
+
+    <div class="spacer-3rem"></div>
+    <div class="spacer-3rem"></div>
+    <div class="spacer-3rem"></div>
+    <div class="spacer-3rem"></div>
+
+    <!-- main -->
+    <main>
+      <!-- featured brands -->
+      <h3 class="h3 font-semibold text-center text-slate-900 mb-1">
+        Featured Brands
+      </h3>
+      <!-- <div class="w-96 bg-rose-500 h-1 m-auto"></div> -->
+      <div class="spacer-2rem"></div>
+      <div class="spacer-2rem"></div>
+
+      <div
+        class="grid grid-cols-4 w-full m-auto"
+        style="max-width: 60rem; gap: 1rem"
+      >
+        <div
+          class="h-52 w-52 text-white rounded-2xl cursor-pointer flex justify-center align-items-center home__page__brand__logo"
+          style="background-image: url(/assets/crayola-logo.png)"
+        ></div>
+        <div
+          class="h-52 w-52 text-white rounded-2xl cursor-pointer flex justify-center align-items-center home__page__brand__logo"
+          style="background-image: url(/assets/faber-castell-logo.png)"
+        ></div>
+        <div
+          class="h-52 w-52 text-white rounded-2xl cursor-pointer flex justify-center align-items-center home__page__brand__logo"
+          style="background-image: url(/assets/staedtler-logo.png)"
+        ></div>
+        <div
+          class="h-52 w-52 text-white rounded-2xl cursor-pointer flex justify-center align-items-center home__page__brand__logo"
+          style="background-image: url(/assets/Maped-logo.png)"
```

Comment:
Please try to avoid inline css. Doing this through a CSS class would be much better and more readable.